src/ralph/virtual/management/commands/openstack_sync.py

```diff
@@ -57,11 +57,16 @@ def __str__(self):
         repr(self.value)
 
 
+class Summary(defaultdict):
+    def __getitem__(self, key):
+        return dict.get(self, key, 0)
```

Comment:
what's the difference between this and just `defaultdict(int)`?

Reply:
It returns 0 if key does not exist instead of raising KeyError

Reply:
Yes, but thats how defaultdict works:

```
>>> from collections import defaultdict
>>> d = defaultdict(int)
>>> d['non-existing-key']
0
```